frontend/public/kubevirt/components/nav.jsx

```diff
@@ -38,7 +38,7 @@ const PageNav = ({ onNavSelect, ResourceClusterLink, HrefLink, ResourceNSLink, M
 
       <NavSection title="Workloads">
         <ResourceNSLink resource="virtualmachines" name="Virtual Machines" />
-        <ResourceNSLink resource="vmtemplates" name={VmTemplatesPageTitle} />
```

Comment:
as @suomiy said, this could bring issues once (if) general templates page is created.
We could however keep `resource="vmtemplates"` and change `VmTemplateModel` in `vm.ts` to

```
export const VmTemplateModel: K8sKind = {
  label: 'Template',
  labelPlural: 'Templates',
  apiVersion: 'v1',
  path: 'templates',
  apiGroup: 'template.openshift.io',
  plural: 'vmtemplates',
  namespaced: true,
  abbr: 'VMT',
  kind: 'Template',
  id: 'vmtemplate',
  selector: {
    matchLabels: {[TEMPLATE_TYPE_LABEL]: 'vm'},
  },
};
```

ResourceNSLink is matching `resource` prop to model's `plural`

Reply:
Done!